inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongGroupTransfer.java

```diff
@@ -49,20 +50,35 @@
 
 public class InlongGroupTransfer {
 
-    public static InlongGroupConf parseGroupResponse(InlongGroupResponse groupResponse) {
+    public static InlongGroupConf parseGroupResponse(InlongGroupResponse inlongGroupResponse) {
         InlongGroupConf inlongGroupConf = new InlongGroupConf();
-        inlongGroupConf.setGroupName(groupResponse.getName());
-        inlongGroupConf.setDescription(groupResponse.getDescription());
-        inlongGroupConf.setCnName(groupResponse.getCnName());
-        inlongGroupConf.setZookeeperEnabled(groupResponse.getZookeeperEnabled() == 1);
-        inlongGroupConf.setDailyRecords(Long.valueOf(groupResponse.getDailyRecords()));
-        inlongGroupConf.setPeakRecords(Long.valueOf(groupResponse.getPeakRecords()));
-        inlongGroupConf.setMqBaseConf(parseMqBaseConf(groupResponse));
-        inlongGroupConf.setSortBaseConf(parseSortBaseConf(groupResponse));
-        inlongGroupConf.setProxyClusterId(groupResponse.getProxyClusterId());
+        inlongGroupConf.setGroupName(inlongGroupResponse.getName());
+        inlongGroupConf.setDescription(inlongGroupResponse.getDescription());
+        inlongGroupConf.setCnName(inlongGroupResponse.getCnName());
+        inlongGroupConf.setLightWeight(isLightGroup(inlongGroupResponse));
+        inlongGroupConf.setZookeeperEnabled(inlongGroupResponse.getZookeeperEnabled() == 1);
+        inlongGroupConf.setDailyRecords(Long.valueOf(inlongGroupResponse.getDailyRecords()));
+        inlongGroupConf.setPeakRecords(Long.valueOf(inlongGroupResponse.getPeakRecords()));
+        inlongGroupConf.setMqBaseConf(parseMqBaseConf(inlongGroupResponse));
+        inlongGroupConf.setSortBaseConf(parseSortBaseConf(inlongGroupResponse));
+        inlongGroupConf.setProxyClusterId(inlongGroupResponse.getProxyClusterId());
         return inlongGroupConf;
     }
 
+    public static boolean isLightGroup(InlongGroupResponse inlongGroupResponse) {
+        List<InlongGroupExtInfo> extInfos = inlongGroupResponse.getExtList();
+        if (CollectionUtils.isEmpty(extInfos)) {
+            return false;
+        }
+        for (InlongGroupExtInfo extInfo : extInfos) {
```

Comment:
From my side, the mode of lightweight is an important config, maybe we can add this param into InlongGroup?